apps/pollbook/backend/src/local_store.test.ts

```diff
@@ -151,6 +152,33 @@ test('findVotersWithName works as expected - voters without name changes', () =>
       expect.objectContaining({ voterId: voters[5].voterId }),
     ])
   );
+
+  // Test period chars
+  expect(
+    localStore.findVotersWithName({
+      firstName: 'Dy.lan',
+      lastName: '.obrien',
+      middleName: 'mid.',
+      suffix: 'i.',
+    })
+  ).toEqual(
+    expect.arrayContaining([
+      expect.objectContaining({ voterId: voters[0].voterId }),
+    ])
+  );
+
+  expect(
+    localStore.findVotersWithName({
+      firstName: 'Mont',
+      lastName: 'StMichel',
+      middleName: '',
+      suffix: '',
+    })
+  ).toEqual(
+    expect.arrayContaining([
+      expect.objectContaining({ voterId: voters[6].voterId }),
+    ])
+  );
 });
```

Comment:
The testing here is for `findVotersWithName` but if it's a manual search, we go down the `searchVoters` code path. Do both benefit from this change?

Reply:
Yes, `toPatternStartsWith` is used by `searchVoters` and `toPatternMatches` is used by `findVotersWithName`. Both were refactored to use a shared set of ignored punctuation. But you make a good point that we should also test `searchVoters`.